account_invoice_report_commercial_address/README.rst

```diff
@@ -0,0 +1,93 @@
+=========================================
+Account Invoice Report Commercial Address
+=========================================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:04564d0de400567ed5175e5581f61c70adafa56a47c87a844bc17fce853f0df6
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-sygel--technology%2Fsy--account--invoice--reporting-lightgray.png?logo=github
+    :target: https://github.com/sygel-technology/sy-account-invoice-reporting/tree/16.0/account_invoice_report_commercial_address
+    :alt: sygel-technology/sy-account-invoice-reporting
+
+|badge1| |badge2| |badge3|
+
+Display the commercial entity on invoices if the billing contact is
+different from the parent company.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+Simply install the module.
+
+Configuration
+=============
+
+Default Settings
+
+- The "Show Commercial Entity" option is enabled by default.
+- The commercial address displayed corresponds to the parent address of
+  the company.
+
+Usage
+=====
+
+Locate the Setting
+
+- Open an invoice and go to the "Other Information" tab.
+
+Toggle the Checkbox
+
+- You will see a checkbox labeled "Show Commercial Entity", which is
```

Comment:
This value is not active by default, it is computed.

Reply:
@Anxo82 This has not been solved. The "Show Commercial Entity" field is not active by default, it is a computed field that will be active or not depending on other values. Please, explain that.